Added M class and MX and fixed a few related items.

ny4i opened the pull request:
Changed ADIF name to WFD.
Added M class.
Added support for receiving MX as a section for Mexico. Also fixed #484.

I also found an error in TREE LooksLikeAGrid. Perhaps something was not committed. The line pAE := ActiveExchange was of the wrong type but it was never again referenced so I commented it out.

In fixing #484, I also reformatted the band/mode so the columns aligned better. It would be good to check ion a few other contests too.

n4af commented:
Hi Tom -

Any chance you built against 4.118.1  ?
I cannot merge trdos/tree.pas and your copy appears to be .1 rather than .2
??
---
 git checkout WFD-2022-rule-changes
error: Your local changes to the following files would be overwritten by
checkout:
        tr4w/src/trdos/tree.pas

------------------
diff --git a/tr4w/src/trdos/tree.pas b/tr4w/src/trdos/tree.pas
index 9916886..10359c9 100644
--- a/tr4w/src/trdos/tree.pas
+++ b/tr4w/src/trdos/tree.pas
@@ -3293,11 +3293,11 @@ function LooksLikeAGrid(var GridString:
ShortString): boolean;
 var
   TestString                            : Str20;
   i                                     : integer;
-  pAE                                   : pchar;
+  p                                  : pchar;^M
 begin

   LooksLikeAGrid := False;
-  pAE := ActiveExchange;
+ ^M
   TestString := UpperCase(GridString);
     if          ((ActiveExchange = RSTAndOrGridExchange) or        //
4.118.1
                 (ActiveExchange = Grid2Exchange) or

On Tue, Dec 27, 2022 at 10:03 PM Tom Schaefer ***@***.***>
wrote:

> Changed ADIF name to WFD.
> Added M class.
> Added support for receiving MX as a section for Mexico. Also fixed #484
> <https://github.com/n4af/TR4W/issues/484>.
>
> I also found an error in TREE LooksLikeAGrid. Perhaps something was not
> committed. The line pAE := ActiveExchange was of the wrong type but it was
> never again referenced so I commented it out.
>
> In fixing #484 <https://github.com/n4af/TR4W/issues/484>, I also
> reformatted the band/mode so the columns aligned better. It would be good
> to check ion a few other contests too.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/n4af/TR4W/pull/630
> Commit Summary
>
>    - 53391e2
>    <https://github.com/n4af/TR4W/pull/630/commits/53391e2d8d3abba4f629f3ed20dee38cebbf977e>
>    Added M class and MX and fixed a few related items.
>
> File Changes
>
> (4 files <https://github.com/n4af/TR4W/pull/630/files>)
>
>    - *M* tr4w/src/VC.pas
>    <https://github.com/n4af/TR4W/pull/630/files#diff-096130b1a19f941e43f0256126996e6cc6860bf635143520357e6046364a8e64>
>    (2)
>    - *M* tr4w/src/trdos/LOGSTUFF.PAS
>    <https://github.com/n4af/TR4W/pull/630/files#diff-a29260bf4df1d82b6cc8089d91b47071ba2c06a27c2e607dacba65d64b2645e6>
>    (6)
>    - *M* tr4w/src/trdos/PostUnit.PAS
>    <https://github.com/n4af/TR4W/pull/630/files#diff-9e754f5d51957764ed18ad57397b6b9bea6b13081853f22902c4016820c53953>
>    (34)
>    - *M* tr4w/src/trdos/tree.pas
>    <https://github.com/n4af/TR4W/pull/630/files#diff-72d3e2ea1ab59fcbc51cd50758af28e45eb0d4be2864955e8ff7362bd3d7fb54>
>    (2)
>
> Patch Links:
>
>    - https://github.com/n4af/TR4W/pull/630.patch
>    - https://github.com/n4af/TR4W/pull/630.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/n4af/TR4W/pull/630>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABVLCUALAHTIXJTRVV3WMBDWPOUZHANCNFSM6AAAAAATK4O4AU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

ny4i commented:
I pulled the latest from master and started there. Did you have the latest in master?Principal Solutions ArchitectBetter Software Solutions, Inc. 727-437-2771On Dec 28, 2022, at 7:41 AM, Howie Hoyt ***@***.***> wrote:﻿
Hi Tom -

Any chance you built against 4.118.1  ?
I cannot merge trdos/tree.pas and your copy appears to be .1 rather than .2
??
---
 git checkout WFD-2022-rule-changes
error: Your local changes to the following files would be overwritten by
checkout:
        tr4w/src/trdos/tree.pas

------------------
diff --git a/tr4w/src/trdos/tree.pas b/tr4w/src/trdos/tree.pas
index 9916886..10359c9 100644
--- a/tr4w/src/trdos/tree.pas
+++ b/tr4w/src/trdos/tree.pas
@@ -3293,11 +3293,11 @@ function LooksLikeAGrid(var GridString:
ShortString): boolean;
 var
   TestString                            : Str20;
   i                                     : integer;
-  pAE                                   : pchar;
+  p                                  : pchar;^M
 begin

   LooksLikeAGrid := False;
-  pAE := ActiveExchange;
+ ^M
   TestString := UpperCase(GridString);
     if          ((ActiveExchange = RSTAndOrGridExchange) or        //
4.118.1
                 (ActiveExchange = Grid2Exchange) or

On Tue, Dec 27, 2022 at 10:03 PM Tom Schaefer ***@***.***>
wrote:

> Changed ADIF name to WFD.
> Added M class.
> Added support for receiving MX as a section for Mexico. Also fixed #484
> <https://github.com/n4af/TR4W/issues/484>.
>
> I also found an error in TREE LooksLikeAGrid. Perhaps something was not
> committed. The line pAE := ActiveExchange was of the wrong type but it was
> never again referenced so I commented it out.
>
> In fixing #484 <https://github.com/n4af/TR4W/issues/484>, I also
> reformatted the band/mode so the columns aligned better. It would be good
> to check ion a few other contests too.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/n4af/TR4W/pull/630
> Commit Summary
>
>    - 53391e2
>    <https://github.com/n4af/TR4W/pull/630/commits/53391e2d8d3abba4f629f3ed20dee38cebbf977e>
>    Added M class and MX and fixed a few related items.
>
> File Changes
>
> (4 files <https://github.com/n4af/TR4W/pull/630/files>)
>
>    - *M* tr4w/src/VC.pas
>    <https://github.com/n4af/TR4W/pull/630/files#diff-096130b1a19f941e43f0256126996e6cc6860bf635143520357e6046364a8e64>
>    (2)
>    - *M* tr4w/src/trdos/LOGSTUFF.PAS
>    <https://github.com/n4af/TR4W/pull/630/files#diff-a29260bf4df1d82b6cc8089d91b47071ba2c06a27c2e607dacba65d64b2645e6>
>    (6)
>    - *M* tr4w/src/trdos/PostUnit.PAS
>    <https://github.com/n4af/TR4W/pull/630/files#diff-9e754f5d51957764ed18ad57397b6b9bea6b13081853f22902c4016820c53953>
>    (34)
>    - *M* tr4w/src/trdos/tree.pas
>    <https://github.com/n4af/TR4W/pull/630/files#diff-72d3e2ea1ab59fcbc51cd50758af28e45eb0d4be2864955e8ff7362bd3d7fb54>
>    (2)
>
> Patch Links:
>
>    - https://github.com/n4af/TR4W/pull/630.patch
>    - https://github.com/n4af/TR4W/pull/630.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/n4af/TR4W/pull/630>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABVLCUALAHTIXJTRVV3WMBDWPOUZHANCNFSM6AAAAAATK4O4AU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

n4af commented:
I think some of 4.118.2 was not committed.
Best I can see, this <https://tr4w.net/4.118/tr4w_setup_4_118.3.exe>picks
up M class and MX section.


On Wed, Dec 28, 2022 at 9:19 AM Tom Schaefer ***@***.***>
wrote:

> I pulled the latest from master and started there. Did you have the latest
> in master?Principal Solutions ArchitectBetter Software Solutions,
> Inc. 727-437-2771On Dec 28, 2022, at 7:41 AM, Howie Hoyt ***@***.***>
> wrote:﻿
> Hi Tom -
>
> Any chance you built against 4.118.1 ?
> I cannot merge trdos/tree.pas and your copy appears to be .1 rather than .2
> ??
> ---
> git checkout WFD-2022-rule-changes
> error: Your local changes to the following files would be overwritten by
> checkout:
> tr4w/src/trdos/tree.pas
>
> ------------------
> diff --git a/tr4w/src/trdos/tree.pas b/tr4w/src/trdos/tree.pas
> index 9916886..10359c9 100644
> --- a/tr4w/src/trdos/tree.pas
> +++ b/tr4w/src/trdos/tree.pas
> @@ -3293,11 +3293,11 @@ function LooksLikeAGrid(var GridString:
> ShortString): boolean;
> var
> TestString : Str20;
> i : integer;
> - pAE : pchar;
> + p : pchar;^M
> begin
>
> LooksLikeAGrid := False;
> - pAE := ActiveExchange;
> + ^M
> TestString := UpperCase(GridString);
> if ((ActiveExchange = RSTAndOrGridExchange) or //
> 4.118.1
> (ActiveExchange = Grid2Exchange) or
>
> On Tue, Dec 27, 2022 at 10:03 PM Tom Schaefer ***@***.***>
> wrote:
>
> > Changed ADIF name to WFD.
> > Added M class.
> > Added support for receiving MX as a section for Mexico. Also fixed #484
> > <https://github.com/n4af/TR4W/issues/484>.
> >
> > I also found an error in TREE LooksLikeAGrid. Perhaps something was not
> > committed. The line pAE := ActiveExchange was of the wrong type but it
> was
> > never again referenced so I commented it out.
> >
> > In fixing #484 <https://github.com/n4af/TR4W/issues/484>, I also
> > reformatted the band/mode so the columns aligned better. It would be good
> > to check ion a few other contests too.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/n4af/TR4W/pull/630
> > Commit Summary
> >
> > - 53391e2
> > <
> https://github.com/n4af/TR4W/pull/630/commits/53391e2d8d3abba4f629f3ed20dee38cebbf977e
> >
> > Added M class and MX and fixed a few related items.
> >
> > File Changes
> >
> > (4 files <https://github.com/n4af/TR4W/pull/630/files>)
> >
> > - *M* tr4w/src/VC.pas
> > <
> https://github.com/n4af/TR4W/pull/630/files#diff-096130b1a19f941e43f0256126996e6cc6860bf635143520357e6046364a8e64
> >
> > (2)
> > - *M* tr4w/src/trdos/LOGSTUFF.PAS
> > <
> https://github.com/n4af/TR4W/pull/630/files#diff-a29260bf4df1d82b6cc8089d91b47071ba2c06a27c2e607dacba65d64b2645e6
> >
> > (6)
> > - *M* tr4w/src/trdos/PostUnit.PAS
> > <
> https://github.com/n4af/TR4W/pull/630/files#diff-9e754f5d51957764ed18ad57397b6b9bea6b13081853f22902c4016820c53953
> >
> > (34)
> > - *M* tr4w/src/trdos/tree.pas
> > <
> https://github.com/n4af/TR4W/pull/630/files#diff-72d3e2ea1ab59fcbc51cd50758af28e45eb0d4be2864955e8ff7362bd3d7fb54
> >
> > (2)
> >
> > Patch Links:
> >
> > - https://github.com/n4af/TR4W/pull/630.patch
> > - https://github.com/n4af/TR4W/pull/630.diff
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/n4af/TR4W/pull/630>, or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ABVLCUALAHTIXJTRVV3WMBDWPOUZHANCNFSM6AAAAAATK4O4AU
> >
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
>
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are
> receiving this because you authored the thread.Message ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/n4af/TR4W/pull/630#issuecomment-1366686904>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABVLCUE4O7246NPUHDHOHLTWPRD5PANCNFSM6AAAAAATK4O4AU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
